app/views/not_found.scala.html

```diff
@@ -33,6 +33,6 @@
             location = appConfig.customsFinancialsFrontendHomepage,
             linkClass = "govuk-link govuk-link--no-visited-state",
             preLinkMessage = Some("cf.error.not-found.message.go-to-home-page-text1"),
-            postLinkMessage = Some("cf.error.not-found.message.go-to-home-page-text2")
+            linkSentence = true
```

Comment:
Is variable defined in the component

Reply:
yes this is defined and it has default value of false